transform/mattermost-analytics/models/intermediate/product/nps/int_nps_feedback.sql

```diff
@@ -0,0 +1,33 @@
+with mattermost_nps as (
+select 
+    distinct server_id as server_id
+    , user_id as user_id
+    , license_id as license_id
+    , event_date as event_date
+    , timestamp as timestamp
+    , server_version as server_version
+    , feedback as feedback
+    , user_role as user_role
+    , received_at as feedback_received_at
+    , CASE WHEN license_id = '{{ var('cloud_license_id') }}' THEN 'Cloud' 
```

Comment:
This information is already available in `dim_server_info`. Perhaps it makes sense to avoid duplicating the work here and work on improving `dim_server_info`.

Reply:
I don't know how to solve this at the moment, since we cannot join `dim` tables with one another. I will need to incorporate this logic from `dim_server_info` into this model, which doesn't apply to these models

```
 -- Aggregate is cloud in order to validate whether the server is cloud or not
count_if(is_cloud = true) as count_is_cloud_days,
count_if(is_cloud = false) as count_not_is_cloud_days,
min(activity_date) as first_activity_date,
max(activity_date) as last_activity_date
```

Reply:
Updated to fact type model to fetch information from `dim_server_info`

Reply:
This no longer applies, closing this as the models only show NPS related data.